python-frontend/src/test/java/org/sonar/python/semantic/ClassSymbolTest.java

```diff
@@ -20,112 +20,225 @@
 package org.sonar.python.semantic;
 
 import org.junit.Test;
+import org.sonar.plugins.python.api.symbols.ClassSymbol;
 import org.sonar.plugins.python.api.symbols.Symbol;
-import org.sonar.plugins.python.api.symbols.Usage;
 import org.sonar.plugins.python.api.tree.ClassDef;
 import org.sonar.plugins.python.api.tree.FileInput;
-import org.sonar.plugins.python.api.tree.FunctionDef;
-import org.sonar.plugins.python.api.tree.Tree;
-import org.sonar.python.PythonTestUtils;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.tuple;
-
+import static org.sonar.python.PythonTestUtils.parse;
 
 public class ClassSymbolTest {
+
+  @Test
+  public void no_parents() {
+    FileInput fileInput = parse(
+      "class C: ",
+      "  pass");
+    assertThat(fileInput.globalVariables()).hasSize(1);
```

Comment:
I don't understand the purpose of this test. Shouldn't you assert that `fileInput.globalVariables()` has 1 symbol of kind `CLASS` with empty parents property?